### ISSUE TITLE
Add Blauth grading thumb hypoplasia

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10463]
**Preferred term label:**
Thumb hypoplasia grade 4

**Synonyms**
Hypoplasia of the thumb grade IV, floating thumb

**Definition (free text, please give PubMed ID)**
Attachment to the hand by the skin and digital neurovascular structures
PMID: 24362256

**Parent term (use hpo.jax.org/app)**
Short thumb HP:0009778

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
See the disease associations from Short thumb HP:0009778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000942
- label: Thumb hypoplasia grade 4
- parents: [Short thumb (HP:0009778)](https://hpo.jax.org/app/browse/term/HP:0009778)
- pmid: PMID:24362256
- def: A type of developmental undergrowth of the thumb characterized by attachment to the hand only by the skin and digital neurovascular structures.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10463
- synonyms: Floating thumb(NONE)|Hypoplasia of the thumb grade IV(NONE)
